src/openrct2-ui/windows/Ride.cpp

```diff
@@ -1201,6 +1201,10 @@ static constexpr const rct_window_graphs_y_axis window_graphs_y_axi[] = {
     {13, -4, 1,  STR_RIDE_STATS_G_FORCE_FORMAT},    // GRAPH_LATERAL
 };
 
+#define RIDE_G_FORCES_RED_POS_VERTICAL FIXED_2DP(5,00)
+#define RIDE_G_FORCES_RED_NEG_VERTICAL -FIXED_2DP(2,00)
+#define RIDE_G_FORCES_RED_LATERAL FIXED_2DP(2,80)
```

Comment:
It is better to use C++ constants now, e.g.:
`static constexpr auto RIDE_G_FORCES_RED_LATERAL = FIXED_2DP(2,80);`

Reply:
Fixed